docs/examples/exemplars/semantic_exemplars.py

```diff
@@ -0,0 +1,84 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Nit: Can we rename `basic_meter` to metrics and put this folder under that in the examples?